CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - targeted/cve-2021-22214.js > A targeted script to check for Unauthorised SSRF on GitLab - CVE 2021-22214
 - httpsender/full-session-n-csrf-nashorn.js > full session and csrf token management.
 - httpfuzzerprocessor/unexpected_responses.js > compare response codes to a (pass/fail) regex and generate alerts
+- targeted/dns-email-spoofing > Check if DMARC / SPF policies are configured on a domain.
```

Comment:
As thc202 mentioned if you move this before one of the existing entries it won't end-up in conflict again if another PR is merged first.

Reply:
Merged, the other PR still has conflicts.